README.md

```diff
@@ -60,6 +60,9 @@ you're in North America or Europe.
 
 Please get in [touch](mailto:jobs@coder.com) with your resume/github if interested.
 
+We're also hiring someone specifically to help maintain code-server.
```

Comment:
No need to put `also` here IMO.

Reply:
Doesn't flow otherwise. Above we're talking generally but here we're mentioning that we specifically want someone for code-server as well.